insights/parsers/freeipa_healthcheck_log.py

```diff
@@ -0,0 +1,69 @@
+"""
+freeipa_healthcheck_log - File ``/var/log/ipa/healthcheck/healthcheck.log``
+===========================================================================
+
+This module provides plugins access to file ``/var/log/ipa/healthcheck/healthcheck.log``
+
+This file contains a list of items, each item representing a check made on the local IPA server.
+The file can be either single-line or multi-line indented.
+
+Typical content of file ``/var/log/ipa/healthcheck/healthcheck.log`` is::
+
+    [
+      {
+        "source": "ipahealthcheck.meta.services",
+        "check": "certmonger",
+        "result": "SUCCESS",
+        "uuid": "ceeae285-6a98-498c-89f9-38cc60279e28",
+        "when": "20190719155725Z",
+        "duration": "0.014892",
+        "kw": {
+          "status": true
+        }
+      },
+      {
+        "source": "ipahealthcheck.meta.services",
+        "check": "dirsrv",
+        "result": "SUCCESS",
+        "uuid": "b5faf965-2df9-4b3d-b6d3-0b29bbf293a5",
+        "when": "20190719155725Z",
+        "duration": "0.013207",
+        "kw": {
+          "status": true
+        }
+      }
+    ]
+
+The list of errors can be accessed via the errors property.
+Examples:
```

Comment:
You need to add a blank line above `Examples:` so that it will render properly.

Reply:
Solved, thanks